### PR TITLE
Backend/CreateChangeLogForDatabaseStructure

### LINKED ISSUE #186: Create changeLog for structure for existing database

### PR DESCRIPTION
**Pull request by SaneQQQQ**

### CODE REVIEW

File: src/main/resources/db/changelog/start_database_structure_changelog.yaml [hunk 1]
```diff
@@ -0,0 +1,937 @@
+databaseChangeLog:
+- changeSet:
+    id: 1642775490693-1
+    author: SaneQQQQ (generated)
```
Comment (dshma) [inline src/main/resources/db/changelog/start_database_structure_changelog.yaml]: there is no need to keep the (generated) part, let's get rid of it

**Review by dshma — CHANGES_REQUESTED** [review]
I'd also rename the file to **initial-schema** instead of _start_database_structure_.

**Comment by dshma** [comment]
BTW:
I haven't seen any sequence being created as part of the initial schema, which is a bit queer. Have you tried applying this changelog to reflect a brand new schema?

**Comment by SaneQQQQ** [comment]
> BTW: I haven't seen any sequence being created as part of the initial schema, which is a bit queer. Have you tried applying this changelog to reflect a brand new schema?

Yes, I find the difference between restore file and generated initial schema. Now I create all the sequences and add it to fields as default value.

**Review by dshma — APPROVED** [review]

**Pull request merged**